### PR TITLE
Reduced duplicated code after gets/cas code merge

### PR DESCRIPTION
**Pull request by argaen**
After introducing gets/cas commands, multi_get function code was somehow duplicated. With this MR both functions are merged to only one.

### CODE REVIEW

File: aiomcache/client.py [hunk 1]
```diff
@@ -213,7 +170,8 @@ def multi_get(self, conn, *keys):
         :raises:``ValidationException``, ``ClientException``,
         and socket errors
         """
-        return (yield from self._multi_get(conn, *keys))
+        values, _ = yield from self._multi_get(conn, *keys)
+        return [values.get(key) for key in keys]
```
Comment (fafhrd91) [inline aiomcache/client.py]: lets return tuple

Comment (argaen) [inline aiomcache/client.py]: good catch, fixed

**Comment by argaen** [comment]
Mmm apparently multi_get should return list according to the test:

```python
    test_value = yield from mcache.multi_get(key1, key2)
    assert test_value == [value1, value2]

    test_value = yield from mcache.multi_get(b'not' + key1, key2)
    assert test_value == [None, value2]
    test_value = yield from mcache.multi_get()
    assert test_value == []
```

**Comment by fafhrd91** [comment]
It is fine to modify test

**Comment by fafhrd91** [comment]
Lgtm

**Comment by argaen** [comment]
Do I merge myself or we have to wait for another reviewer or?

**Comment by fafhrd91** [comment]
Merge yourself, you should have permission

**Comment by argaen** [comment]
Should we create a new release with the feature? I'm planning on using it for https://github.com/argaen/aiocache

**Comment by fafhrd91** [comment]
sure, feel free to create new release.

**Comment by fafhrd91** [comment]
though #13 actually a problem.

**Comment by argaen** [comment]
@fafhrd91 I don't have permissions for uploading the new release to pypi

**Comment by fafhrd91** [comment]
it should happen automatically

**Comment by fafhrd91** [comment]
travis build is not completed yet https://travis-ci.org/aio-libs/aiomcache/builds/199782749

**Pull request merged**